src/utils/positionpnl/positionpnl.js

```diff
@@ -0,0 +1,64 @@
+import {
+  retrieveData,
+  insertData,
+  putData
+} from '../network'
+import PositionPNL from '../../transactions/PostionPNL/PositionPNL'
+
+export function retrieve({ AMId }, callback) {
+  const params = {
+    AMaaSClass: 'positionpnl',
+    AMId
+  }
+  let promise = retrieveData(params)
+    .then(result => {
+      result = result.map(positionpnl => _parsePositionPNL(pnl))
+      if (typeof callback === 'function') {
+        callback(null, result)
+      }
+      return result
+    })
+    if (typeof callback !== 'function') {
+      return promise
+    }
+}
+
+export function amend({ AMId }, callback) {
```

Comment:
Does this need any other arguments? Like what the amendments are. You can check the `putData` function to see what it accepts as queries/body.